08-projects.Rmd

```diff
@@ -7178,10 +7188,12 @@ sqlite3 /class/datamine/data/lahman/lahman.db
 **Item(s) to submit:**
 
 - SQL code used to solve the problem.
-- The first 10 results of the query.
+- The team-year combination that ranked top in each category.
 ```
 
-##### 9. Get a list of the following by year: wins (`W`), losses (`L`), Home Runs Hit (`HR`), homeruns allowed (`HRA`), and attendance for the Detroit Tigers when appearing in a World Series (`WSWin`) or when league champion (`LgWin`).
+##### 9. Get a list of the following by year: wins (`W`), losses (`L`), Home Runs Hit (`HR`), homeruns allowed (`HRA`), and total home game attendance (`attendance`) for the Detroit Tigers when appearing in a World Series (`WSWin`) or when league champion (`LgWin`).
```

Comment:
I added the clarification for "attendance": I first thought it is something related to how many players participated in a game.